docs/template_plugin/tests/functional/op_reference/irdft.cpp

```diff
@@ -450,6 +484,47 @@ static const std::vector<float> expected_rdft3d_results_2 = {
+    0.00807303,   0.02364844,   -0.00346402,   -0.00134415,   0.04106979,   0.04961361,
+    -0.01212564,   -0.04288128,   -0.26157875,   -0.01917418,   -0.04232584,   0.02477720,
+    0.02514449,   0.04955597,   -0.00301304,   0.00663580,   0.01947190,   -0.01163269,
+    -0.07920224,   -0.01201069,   0.00564843,   0.00283007,   -0.05916596,   0.03569793,
+    -0.02454099,   -0.01977048,   -0.00360401,   0.00924050,   -0.01237082,   -0.04213287,
+    -0.03306797,   -0.01442351,   -0.02601594,   0.07406829,   -0.02896844,   0.00503278,
+    0.00700455,   0.02915976,   0.01761130,   -0.04474307,   0.03632101,   0.00957998,
+    -0.02003984,   -0.04022581,   0.03104216,   0.00388626,   0.05861915,   0.01034101,
+    -0.00741989,   0.01010181,   0.01496502,   -0.00544559,   0.04015258,   -0.00600315,
+    -0.06137903,   0.07850411,   -0.00074931,   0.02540785,   -0.00166176,   0.02205904,
+    -0.02429718,   0.04010517,   0.02375359,   0.02229406,   0.01806382,   -0.06089136,
+    0.00447113,   -0.03169147,   0.02836490,   -0.05821620,   0.03905417,   0.03987032,
+    0.29899586,   -0.02616866,   -0.00927641,   -0.02134532,   -0.02480746,   -0.02636082,
+    -0.05009444,   -0.02208490,   0.02632000,   0.00493334,   -0.00402312,   -0.00935831,
+    0.04154630,   0.00849218,   0.00232782,   -0.01192997,   -0.03309486,   0.01678531,
+    0.03526979,   0.09272132,   0.01420703,   -0.01919909,   0.01321082,   -0.01661140,
+    0.07861365,   -0.02784724,   0.03900426,   -0.00096805,   -0.02880604,   0.02753764,
+    -0.02092520,   -0.01412453};
+
+static const std::vector<float> expected_rdft3d_results_4 = {
+    0.24882269,   -0.00554157,   -0.00759689,   -0.00413212,   0.01099624,   0.02191469,
+    0.02829072,   -0.01410181,   0.04826954,   0.03587530,   -0.01151859,   0.03459743,
+    0.03157633,   -0.03446264,   0.03595825,   -0.01176664,   0.00625817,   0.00981066,
+    -0.11900401,   -0.02756717,   0.01933546,   0.03042892,   -0.04917013,   0.00048474,
+    -0.01849990,   -0.01050222,   -0.02433642,   -0.08657554,   -0.03473007,   -0.01486101,
+    0.00137630,   -0.01972852,   -0.06159696,   0.02284726,   -0.03851998,   -0.00885092,
+    0.02397606,   -0.02071742,   -0.00586151,   -0.01287085,   0.01713095,   -0.07724825,
+    0.05983482,   -0.02824272,   0.02959802,   0.04051825,   0.00219584,   0.04053028,
+    0.00415529,   0.02379833,   -0.01936524,   0.04350142,   0.02095385,   0.03121966,
+    -0.02675550,   0.01142533,   0.05606331,   0.02115209,   0.00866956,   0.05367358,
+    -0.00479556,   0.05423974,   -0.01172735,   -0.01203834,   0.00181946,   0.00594081,
+    0.00527473,   0.00781714,   0.07042868,   -0.02243115,   0.03207793,   -0.04213578,
+    0.14912935,   -0.01012542,   -0.05799989,   -0.02889979,   0.02934662,   0.03385938,
+    0.00951527,   -0.01760542,   -0.01611288,   0.29838892,   -0.01029289,   -0.06226702,
+    -0.03670440,   0.03954893,   0.00725941,   0.04219448,   -0.03698240,   0.03564729};
```

Comment:
I'm just curious, how did you get these results? Do you just fix the ngraph reference implementation and get results from the output? Or do you also run this operation on some other framework, say NumPy, and compare results?

Reply:
Thank you! I was looking for such a template because it's sometimes helpful to compare our results with our library.